src/ProjectService.Business/Commands/Image/CreateImageCommand.cs

```diff
@@ -12,123 +16,125 @@
 using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
 using LT.DigitalOffice.ProjectService.Validation.Interfaces;
 using MassTransit;
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net;
 
 namespace LT.DigitalOffice.ProjectService.Business.Commands.Task
 {
-    public class CreateImageCommand : ICreateImageCommand
+  public class CreateImageCommand : ICreateImageCommand
+  {
+    private readonly IImageRepository _repository;
+    private readonly IRequestClient<ICreateImagesRequest> _rcImages;
+    private readonly ILogger<CreateImageCommand> _logger;
+    private readonly IAccessValidator _accessValidator;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IDbImageMapper _dbProjectImageMapper;
+    private readonly ICreateImageValidator _validator;
+    private readonly IUserRepository _userRepository;
+
+    private List<Guid> CreateImage(List<ImageContent> context, Guid userId, List<string> errors)
     {
-        private readonly IImageRepository _repository;
-        private readonly IRequestClient<ICreateImagesRequest> _rcImages;
-        private readonly ILogger<CreateImageCommand> _logger;
-        private readonly IAccessValidator _accessValidator;
-        private readonly IHttpContextAccessor _httpContextAccessor;
-        private readonly IDbImageMapper _dbProjectImageMapper;
-        private readonly ICreateImageValidator _validator;
-
-        private List<Guid> CreateImage(List<ImageContent> context, Guid userId, List<string> errors)
-        {
-            List<CreateImageData> images = context
-                .Select(x => new CreateImageData(x.Name, x.Content, x.Extension, userId))
-                .ToList();
-
-            string logMessage = $"Errors while creating images for user id {userId}. ";
-
-            try
-            {
-                IOperationResult<ICreateImagesResponse> response = _rcImages.GetResponse<IOperationResult<ICreateImagesResponse>>(
-                   ICreateImagesRequest.CreateObj(images, ImageSource.Project)).Result.Message;
-
-                if (response.IsSuccess && response.Body.ImagesIds != null)
-                {
-                    return response.Body.ImagesIds;
-                }
-
-                _logger.LogWarning(
-                    logMessage + "Errors: { Errors}",
-                    string.Join('\n', response.Errors));
-            }
-            catch (Exception exc)
-            {
-                _logger.LogError(exc, logMessage);
-            }
-
-            errors.Add("Can not create images. Please try again later.");
-
-            return null;
-        }
+      List<CreateImageData> images = context
+        .Select(x => new CreateImageData(x.Name, x.Content, x.Extension, userId))
+        .ToList();
+
+      string logMessage = $"Errors while creating images for user id {userId}.";
```

Comment:
we create images for a project,  not for user

Reply:
done